### DISCUSSION TITLE
Trajectory Timestep

### DISCUSSION THREAD

**Discussion by hblumberg** [Parcels-code/Parcels#1192] [Q&A] [answered]
It seems that the timestep of the outputted trajectories is the minimum of:
* the timestep specified when executing the kernel (the `dt` argument to [execute](https://oceanparcels.org/gh-pages/html/#parcels.particleset.baseparticleset.BaseParticleSet.execute))
* the smallest timestep of the fieldset

Is this correct? Thank you!

**Comment by erikvansebille** [comment, marked as answer]
Yep, that is correct. If `outputdt < dt`, then the particles wouldn't have moved/changed, so it would nog make sense to output again.

**Reply by hblumberg** [reply]
Thanks so much for confirming!
